src/components/Card/Card.tsx

```diff
@@ -64,19 +64,19 @@ const CardComponent: React.FC = () => {
                     alt={activeCard.title}
                     className="w-full h-1/2 object-cover rounded-2xl"
                 />
-                <div className="flex mt-6">
-                    <h2 className="text-4xl flex items-center text-white font-thin mr-6">
+                <div className="flex mt-6 justify-center sm:justify-start">
+                    <h2 className="text-2xl sm:text-4xl flex text-white font-thin mr-2 sm:mr-6 text-center sm:text-start">
                         {activeCard.title}
                     </h2>
                     <ArrowTopRightIcon width={22} height={22} />
                 </div>
             </div>
             {/* Description */}
             <div className="mt-3">
-                <p className="text-sm text-white">{activeCard.description} </p>
+                <p className="text-sm text-white text-center sm:text-start">{activeCard.description} </p>
             </div>
             {/* Buttons */}
-            <div className="flex justify-between mt-10">
+            <div className="flex justify-center sm:justify-between flex-wrap gap-2 sm:gap-0 mt-10">
                 <div className="flex items-center justify-center">
```

Comment:
![image](https://github.com/user-attachments/assets/8259989e-d600-4ab6-9777-c6b652f79f41)

^ on 1024px and bigger breakpoints. maybe:
```suggestion
            <div className="flex flex-col sm:flex-row justify-between items-center gap-4 mt-10">
                <div className="flex items-center order-last sm:order-first">
```

(and then `whitespace-nowrap` on the View Project button)

Reply:
Good catch and W solve @AmirAgassi, thank you!